Modules/Utils/Sources/PocketCastsUtils/Extensions/UI/UIView+distanceToBottom.swift

```diff
@@ -0,0 +1,18 @@
+#if !os(watchOS)
+import UIKit
+
+extension UIView {
+    public func distanceFromBottom() -> CGFloat? {
+        guard let window = self.window else {
+            return nil
+        }
+
+        let viewFrameInWindow = self.convert(self.bounds, to: window)
```

Comment:
Nit: You can avoid `self.` in both places

Reply:
Done in c237ddff541659d483e2ae3837afedf8ad7ef450